src/components/base/translate/Translate.json

```diff
@@ -891,6 +891,28 @@
         "en": "Download file for EndNote, Zotero, Mendeley and ReferenceManager"
       }
     },
+    "multiReferences": {
+      "get-references": {
+        "da": "Hent referencer for %s materialer",
```

Comment:
Hvad hvis det er ental? "Hent reference for materialet" ?

Reply:
så det som en del af "vis man kun vælger et materiale", men måske har du ret og det hører hertil

Reply:
Du har ret